pulp_ci/Containerfile

```diff
@@ -95,4 +95,4 @@ COPY assets/nginx.run /etc/services.d/nginx/run
 
 ENTRYPOINT ["/init"]
 
-EXPOSE 80
+EXPOSE 80 24817
```

Comment:
I'd rather not have this in the base image. This is also used to create the "single container installation" images.

Would it be possible to add `--publish 24817` to the container call in the CI itself?

Reply:
It makes sense, I was publishing the port when creating the container in my CI testing bust wasn't sure if it should have been exposed by default. Will remove the entry here.